service/server_test.go

```diff
@@ -404,6 +410,79 @@ func Test_server_ListenAndServe(t *testing.T) {
 				},
 			}
 		}(),
+		func() test {
+			ctx, cancelFunc := context.WithCancel(context.Background())
+			key := "./assets/dummyServer.key"
+			cert := "./assets/dummyServer.crt"
+
+			handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				w.WriteHeader(200)
+				fmt.Fprintln(w, "Hello, client")
+			})
+
+			apiSrvPort := 9998
+			hcSrvPort := 9999
+			apiSrvAddr := fmt.Sprintf("https://127.0.0.1:%v", apiSrvPort)
+			hcSrvAddr := fmt.Sprintf("http://127.0.0.1:%v", hcSrvPort)
+
+			return test{
+				name: "Test health check server disable",
+				fields: fields{
+					srv: func() *http.Server {
+						srv := &http.Server{
+							Addr:    fmt.Sprintf(":%d", apiSrvPort),
+							Handler: handler,
+						}
+
+						srv.SetKeepAlivesEnabled(true)
+						return srv
+					}(),
+					hcsrv: func() *http.Server {
+						srv := &http.Server{
+							Addr:    fmt.Sprintf(":%d", hcSrvPort),
+							Handler: handler,
+						}
+
+						srv.SetKeepAlivesEnabled(true)
+						return srv
+					}(),
+					cfg: config.Server{
+						Port: apiSrvPort,
```

Comment:
```suggestion
						Port: apiSrvPort,
						// HealthzPort: hcSrvPort,
```
leave a comment? easier to understand the test case

Reply:
added :)